### PR TITLE
Clarify that FLS/DLS disable shard request cache

### PR DESCRIPTION
**Pull request by mayya-sharipova**
Relates to #44581

### CODE REVIEW

**Comment by mayya-sharipova** [comment]
@elasticmachine run elasticsearch-ci/docs

File: x-pack/docs/en/security/authorization/field-and-document-access-control.asciidoc [hunk 1]
```diff
@@ -25,6 +25,9 @@ grant wider access than intended. Each user has a single set of field level and
 document level permissions per index. See <<multiple-roles-dls-fls>>.
 =====================================================================
 
+NOTE: Document and field level security currently disables
```
Comment (jrodewig) [inline x-pack/docs/en/security/authorization/field-and-document-access-control.asciidoc]:
```suggestion
NOTE: Document- and field-level security disables the
```

**Review by jrodewig — APPROVED** [review]
Left a minor comment. LGTM.

File: x-pack/docs/en/security/authorization/field-and-document-access-control.asciidoc [hunk 2]
```diff
@@ -25,6 +25,9 @@ grant wider access than intended. Each user has a single set of field level and
...
+<<shard-request-cache, shard request cache>>.
```
Comment (jrodewig) [inline x-pack/docs/en/security/authorization/field-and-document-access-control.asciidoc]:
```suggestion
{ref}/shard-request-cache.html[shard request cache].
```

**Pull request merged**